src/NServiceBus.Testing.Fakes/TestingLoggerFactory.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace NServiceBus.Testing
+{
+    using System;
+    using System.IO;
+    using Logging;
+
+    /// <summary>
+    /// Logger factory which allows to log to a text writer.
+    /// </summary>
+    public class TestingLoggerFactory : LoggingFactoryDefinition
+    {
+        /// <summary>
+        /// Creates a new instance of a testing logger factory.
+        /// </summary>
+        public TestingLoggerFactory()
+        {
+            level = new Lazy<LogLevel>(() => LogLevel.Debug);
+            writer = new Lazy<TextWriter>(() => new StringWriter());
```

Comment:
If this writer is accessed before `WriteTo` is called, then the new instance of `StringWriter` will not be disposed.  Since this instance of `StringWriter` is created by the `Lazy<>`, this class owns the instance and should manage it's `IDisposable` process.

Something we want to rectify or acceptable leak?